common/changes/@uifabric/experiments/master_2018-06-14-02-39.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Dashboard Card Component for Admin Portals Dashboard which will be hosted inside  react -grid-layout. ",
```

Comment:
>react -grid-layout [](start = 100, length = 18)

why is there extra space?

Reply:
fixed

---
In reply to: [195603935](https://github.com/OfficeDev/office-ui-fabric-react/pull/5196#discussion_r195603935) [](ancestors = 195603935)